__tests__/chat.test.js

```diff
@@ -175,6 +176,60 @@ describe('Chat Methods', () => {
     // it("Throws an error if it cannot react to a message (e.g., it's not a reactable message type")
   })
 
+  describe('Chat attach', () => {
+    const attachmentLocation = '/tmp/kb-attachment.txt'
+    beforeAll(() => {
+      fs.writeFileSync(attachmentLocation, 'This is a test file!')
```

Comment:
I think it'd be better to include `import {promisify} from 'util'` at the top and have all your fs operations async. I don't think blocking the event loop is a good idea. 

something like this, I believe, will get you clean lines without callback hell.

```  
beforeAll(async () => {
    await promisify(fs.writeFile)(attachmentLocation, 'This is a test file!')
  })
```